registry/sql-registry/api-spec.md

```diff
@@ -364,3 +364,62 @@ Create new derived feature in the project
 | Field | Type |
 |-------|------|
 | guid  | Guid |
+
+
+## Access Control
+
+### Access Role Mapping
+
+| Pre-Defined Role   | Access                             |
+| ------------------ | ---------------------------------- |
+| Admin              | Read, Write, Management            |
+| Contributor        | Read, Write                        |
+| Viewer             | Read Only                          |
+| Default            | NA (configurable)                  |
+| Global Admin       | Admin Access to All Projects       |
+| Global Contributor | Contributor Access to All Projects |
+| Global Viewer      | Viewer  Access to All Projects     |
+
++ Rules
+  + Project creator will defaultly get `Admin` access to the Project
+  + When a project is shared, the `Read` access is granted.
+  + In this version, customized role is NOT supported.
+
+### UserRoles
+
+Type: Object
+
+| Field        | Type      |
+| ------------ | --------- |
+| id           | `string`  |
```

Comment:
>string [](http://example.com/codeflow?start=18&length=6)

instead of string, should use guid or auto incremental id

Reply:
Let's use GUID for all entities, auto-inc id can be troublesome sometimes.